File: 02_filter_array.py
#!/usr/bin/env python3

# Python 3.9.5

# 02_filter_prices.py

prices = [15, 13, 9, 19, 7, 4, 21, 12]
prices_in_range = [price for price in prices if price < 10]

print("Filtering prices for prices in range:", prices)
print("Prices in range:", prices_in_range)

File: 02_filter_array.js
// Filter prices from an array to prices within a range
const prices = [15, 13, 9, 19, 7, 4, 21, 12];
const prices_in_range = prices.filter(function(price) {
    return price < 10;
});

document.write("Filtering prices for prices in range:", prices, "<br>");
document.write("Prices in range:", prices_in_range, "<br>");
